view/frontend/web/template/payment/razorpay-form.html

```diff
@@ -4,7 +4,7 @@
                name="payment[method]"
                class="radio"
                data-bind="attr: {'id': getCode()}, value: getCode(), checked: isChecked, click: selectPaymentMethod, visible: isRadioButtonVisible()"/>
-        <label data-bind="attr: {'for': getCode()}" class="label"><span data-bind="text: getTitle()"></span></label>
```

Comment:
@seherkdc Why have we removed this?

Reply:
@ccrims0n The removed line of code displayed "Razorpay" in text. If we're adding the Razorpay wordmark logo, then both text and image become repetitive.

Reply:
Screenshot of only Razorpay Logo
![Screenshot (1)](https://user-images.githubusercontent.com/47523938/57829007-31dacc80-77cb-11e9-8a91-d732663ff5f6.jpg)


Screenshot of Razorpay text with Logo
![Screenshot](https://user-images.githubusercontent.com/47523938/57829000-2c7d8200-77cb-11e9-8fdf-d4b020f91534.jpg)

Reply:
looks good.